src/main/java/catan/domain/transfer/output/game/GameDetails.java

```diff
@@ -33,7 +33,7 @@ public GameDetails(GameBean game, int detailsRequesterId) {
         this.dateCreated = game.getDateCreated().getTime();
         this.dateStarted = game.getDateStarted() != null ? game.getDateStarted().getTime() : 0;
         this.status = game.getStatus().name();
-        this.gameUsers = game.getGameUserDetails(detailsRequesterId);
+        this.gameUsers = game.getGameUserDetails(detailsRequesterId, game.getStatus());
```

Comment:
this is shit, we should not pass in the game's method parameter that we take from game